### PR TITLE
docker: create development environment

### PR DESCRIPTION
**Pull request by otaviopace**
## Description

Create a service to run the application for development.
Also updates version of `eslint-plugin-import`.

## Your checklist for this pull request

:rotating_light: Please review this items for a good pull request. :four_leaf_clover:

1. I've read the project's [Contributing Guidelines](../CONTRIBUTING.md)
1. My commits are well written and follow [pagarme/git-style-guide](https://github.com/pagarme/git-style-guide)
1. My changes are well covered by **tests** and **logs**
1. I've updated the project docs (if needed)
1. I fell safe about this implementation
1. I feel comfortable with the code I wrote, and I'm not ashamed to show it to my friends

In a good pull request, everything above is true :relaxed:

this closes https://github.com/pagarme/ghostbusters/issues/7

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "private": true,
   "scripts": {
-    "dev": "node ./src/bin/server.js",
+    "dev": "npm run setup && node ./src/bin/server.js",
```
Comment (grvcoelho) [inline package.json]: Eu não acho que a gente deveria rodar o setup antes de subir o servidor. Isso deve ser executado antes, sim, mas não por esse script. Talvez numa Makefile ou algo assim.

File: docker-compose.yml [hunk 2]
```diff
@@ -43,3 +44,21 @@ services:
       - /superbowleto/node_modules
     depends_on:
       - test-dependencies
+
+  dev:
```
Comment (grvcoelho) [inline docker-compose.yml]: Chama esse serviço de `web` ou `server` ou `superbowleto-server`. `dev` é muito vago

Comment (marquinhoalm) [reply docker-compose.yml]: Concordo. Acho `server` mais interessante

File: script/setup [hunk 3]
```diff
@@ -1,4 +1,4 @@
-#!usr/bin/env node
+#!/usr/local/bin/node
```
Comment (marquinhoalm) [inline script/setup]: Não conheço como funciona o shell. Isso não vai forçar o path do node certo?

Comment (otaviopace) [reply script/setup]: Ele vai forçar o `node` que tem no container `web-server`

File: README.md [hunk 4]
```diff
@@ -47,9 +47,34 @@ If you never developed in this repo before:
...
-  $ docker-compose build test
+  $ docker-compose build web-server
+  ```
+
+### Running the server
+
+To run the server, you will have to start the database and run the migrations.
+
+1. *Start database and run migrations in one command:*
+    ```sh
```
Comment (grvcoelho) [inline README.md]: Remove one level of indentation.

File: README.md [hunk 5]
```diff
@@ -47,9 +47,34 @@ If you never developed in this repo before:
...
+    $ make setup-db
+    ```
+
+2. *Or start database and run migrations separately:*
+  - **Start database (postgres):**
+    ```sh
```
Comment (grvcoelho) [inline README.md]: Remove one level of indentation.

File: README.md [hunk 6]
```diff
@@ -47,9 +47,34 @@ If you never developed in this repo before:
...
+    $ make start-db
+    ```
+
+  - **Run the migrations:**
+    ```sh
```
Comment (grvcoelho) [inline README.md]: Remove one level of indentation.

**Pull request merged**